### DISCUSSION TITLE
Documentation [Jul 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2700] [Discord Threads]
> 🙋‍♂️ **webInRoad**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### https://docs.nocodb.com/   
#### https://apis.nocodb.com/   
####    
#### Implementation details are not captured at large. We will continue to work on improving documentation & cover as much as possible.  
#  
> 🙋‍♂️ **webInRoad**  
#### Are there any articles or videos related to the technology used in your project?  
#  
> 👨‍🦲 **dstala**  
#### not much articles/videos <@994889556165927033> . project is open source, you can explore here (www.github.com/nocodb/nocodb)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/994892815718944788)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
